### ISSUE TITLE
No pretrained model to be logistic regression will also achieve 83+ on cifar-10

### ISSUE THREAD

**Issue by yanghu819** [Spijkervet/BYOL#11]
before pre-train I save the checkpoint which should be very weak in representation
![image](https://user-images.githubusercontent.com/44990984/219338971-0ebd38f8-695b-4fcc-b016-8a493331084e.png)

and perform logistic regression as:
python3 logistic_regression.py --model_path=./model-no-train.pt
and the result is
Epoch [296/300]: Loss/train: 0.4051439380645752 Accuracy/train: 0.8583268229166667
Epoch [297/300]: Loss/train: 0.4050535774230957 Accuracy/train: 0.8583854166666667
Epoch [298/300]: Loss/train: 0.4049636995792389 Accuracy/train: 0.8583333333333333
Epoch [299/300]: Loss/train: 0.40487425565719604        Accuracy/train: 0.8583528645833333
### Calculating final testing performance ###
Final test performance: Accuracy/test: 0.8356724330357143
